provisioning/device/src/TransportSettings/ProvisioningClientHttpSettings.cs

```diff
@@ -15,5 +15,15 @@ public ProvisioningClientHttpSettings()
         {
             Protocol = ProvisioningClientTransportProtocol.WebSocket;
         }
+
+        internal ProvisioningClientHttpSettings Clone()
+        {
+            return new ProvisioningClientHttpSettings()
```

Comment:
not: parens not needed when using initializer list.

```suggestion
            return new ProvisioningClientHttpSettings
```